<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe | New Paper "Disseminating Metaproteomic Informatics Capabilities and Knowledge Using the Galaxy-P Framework"</title>
  <meta property="og:title" content="New Paper "Disseminating Metaproteomic Informatics Capabi..." />
  <meta property="og:description" content="Clemens and the GalaxyP team have published a new article..." />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/posts/2018/02/01/metaproteomics/plain.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          




<section id="news-page">
  <div class="row news-header-row">
    
    <a class="col-xs-1 text-center" href="/posts/2018/01/30/Gilsbach_NatCom/plain.html">
      «
    </a>
    
    <div class="col-xs-10 ">
      <div class="text-center">
        <h2 class="title">New Paper &quot;Disseminating Metaproteomic Informatics Capabilities and Knowledge Using the Galaxy-P Framework&quot;</h2>
      </div>
      <hr />
    </div>
    
    <a class="col-xs-1 text-center" href="/posts/2018/02/11/genome-annotation-reloaded/plain.html">
      »
    </a>
    
  </div>
  <div class="row post-metadata">
    <div class="col-md-8 col-md-offset-2">
      <div class="row">
        <div class="col-md-6">
            

        </div>
        <div class="col-md-6 text-right">
          <div>
            <span class="post-date">
              Feb 1, 2018
            </span>

            

            <br />
            <span class="post-tags">
                <span class="label label-success">Freiburg</span>
              
                <span class="label label-info">paper</span>
              
              
            </span>

            
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="row">
    <div class="col-md-8 col-md-offset-2">
      <div class="news-content">
        <p>Clemens and the GalaxyP team have published a new article about metaproteomics and Galaxy. <a href="https://www.mdpi.com/2227-7382/6/1/7">Learn more about it</a>.</p>

<h2 id="abstract">Abstract</h2>

<p>The impact of microbial communities, also known as the microbiome, on human health and the environment is receiving increased attention. Studying translated gene products (proteins)
and comparing metaproteomic profiles may elucidate how microbiomes respond to specific environmental stimuli, and interact with host organisms. Characterizing proteins expressed by a
complex microbiome and interpreting their functional signature requires sophisticated informatics tools and workflows tailored to metaproteomics. Additionally, there is a need to
disseminate these informatics resources to researchers undertaking metaproteomic studies, who could use them to make new and important discoveries in microbiome research. The Galaxy
for proteomics platform (Galaxy-P) offers an open source, web-based bioinformatics platform for disseminating metaproteomics software and workflows. Within this platform, we have
developed easily-accessible and documented metaproteomic software tools and workflows aimed at training researchers in their operation and disseminating the tools for more widespread
use. The modular workflows encompass the core requirements of metaproteomic informatics: (a) database generation; (b) peptide spectral matching; (c) taxonomic analysis and (d)
functional analysis. Much of the software available via the Galaxy-P platform was selected, packaged and deployed through an online metaproteomics “Contribution Fest“ undertaken by a
unique consortium of expert software developers and users from the metaproteomics research community, who have co-authored this manuscript. These resources are documented on GitHub and
freely available through the Galaxy Toolshed, as well as a publicly accessible metaproteomics gateway Galaxy instance. These documented workflows are well suited for the training of
novice metaproteomics researchers, through online resources such as the Galaxy Training Network, as well as hands-on training workshops. Here, we describe the metaproteomics tools
available within these Galaxy-based resources, as well as the process by which they were selected and implemented in our community-based work. We hope this description will increase
access to and utilization of metaproteomics tools, as well as offer a framework for continued community-based development and dissemination of cutting edge metaproteomics software.</p>

<div class="multiple-img">
  <img src="/assets/media/proteomes-06-00007-ag.png" width="800px" alt="metaproteomics workflow" />
</div>

<h2 id="credit">Credit</h2>

<embed src="https://www.mdpi.com/2227-7382/6/1/7" width="100%" height="700" type="application/xhtml+xml" />


      </div>
    </div>
  </div>



</section>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>The <a href="/freiburg/people">Freiburg Galaxy Team</a> is headed by Björn Grüning
and is part of the <a href="http://www.bioinf.uni-freiburg.de">Bioinformatics Group
Freiburg</a>, headed by Prof. Dr. Rolf
Backofen.</p>

<p>The team is part of <a href="https://www.denbi.de">de.NBI</a> and
<a href="https://www.elixir-europe.org">ELIXIR</a> and largely funded by
<a href="http://dfg.de/">DFG</a> and <a href="https://www.bmbf.de">BMBF</a>. All content on
this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>,
unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/_posts_plain/2018-02-01-metaproteomics.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                    
                      <li><img class="fa-mastodon" src="/assets/media/mastodon.svg" style="width:18px;height:18px;padding-right:4px;filter:grayscale(100%);-webkit-filter:grayscale(100%);"/><a href="https://bawü.social/@galaxyfreiburg">galaxyfreiburg</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
