BEGIN:VCALENDAR
PRODID:-//jekyll//NONSGML v1.0//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-CALNAME:SARS-CoV-2 Data Analysis and Monitoring with Galaxy
X-WR-TIMEZONE:Europe/Berlin
BEGIN:VEVENT
ORGANIZER;CN=:MAILTO:?subject=SARS-CoV-2%20Data%20Analysis%20and%20Monitoring%20with%20Galaxy
DTSTART;VALUE=DATE:20210809
DTEND;VALUE=DATE:20210812
DTSTAMP;VALUE=DATE:20210809T000000Z
CREATED;VALUE=DATE:20210621T000000Z
UID:2021080920210812@usegalaxy.eu
DESCRIPTION:The goal of this workshop is to build capacity in SARS-CoV-2 data analysis and data management, including data submission to ENA. After the workshop, all participants will be able to upload viral sequencing data, call all variants, create a variety of reports and create consensus alignments.\n\nIt will be a 4-day event introducing scalable and reproducible SARS-CoV-2 data analysis with Galaxy. The sessions will be pre-recorded and provided in advance. During the workshop, there will be live support in chat and live Q&amp;A sessions, in which experts will answer questions.<br />\n\n<div class="row"><div class="col-md-6">    <ul>      <li><strong>WHEN</strong>: August 9-12, 2021</li>      <li><strong>WHO</strong>: Open for everybody, but the target audience is <strong>clinicians</strong> and <strong>researchers</strong> that deal with SARS-CoV-2 sequencing data.</li>      <li><strong>COST</strong>: Free.</li>      <li><strong>FORMAT</strong>: Virtual and asynchronous. All training <strong>sessions will be pre-recorded and provided in advance</strong>.</li>      <li><strong>SUPPORT</strong>:        <ul>          <li><strong>Live support in chat</strong> (Slack Channel), in which experts will answer questions on a peer-to-peer basis.</li>          <li><strong>Real-Time Q&amp;A sessions</strong> on days 2, 3 and 4 (9 am and 4 pm CEST).</li>        </ul>      </li>      <li><strong>INFRASTRUCTURE</strong>: <a href="https://usegalaxy.eu" target="_blank">European Galaxy server</a> and the <a href="https://training.galaxyproject.org" target="_blank">Galaxy Training Material</a>. Both will stay accessible and open after the training.</li>      <li><a href="mailto:contact@usegalaxy.eu"><strong>Contact us</strong></a> if you have questions.</li>    </ul>  </div><div class="col-md-6">    <img src="/assets/media/2021-06-21-sars-cov-2-data-analysis-monitoring-training.png" alt="sars-cov-2-workshop" />\n\n  </div></div><br />\n\n<a href="https://docs.google.com/forms/d/e/1FAIpQLSeADQn0PdzfFppwoOGFAqWQglnKRhj3FNcqpUKggwXsn2PKBg/viewform?usp=sf_link" class="btn btn-primary" style="color: white; width: 100%; background-color: #3a3566;"><strong>Register now</strong></a>\n\n<br />\n\n<h1 id="program--material">Program &amp; Material</h1>This workshop is virtual and asynchronous. Training sessions are pre-recorded and the material provided in advance in the program below (some material is still in preparation: WIP).\n\nWhenever you’re ready to get started, you can access the material by clicking in the program on the different icons:\n\n<ul>  <li><a href="" class="fa fa-slideshare"></a> for <strong>slide</strong> deck for a lecture</li>  <li><a href="" class="fa fa-laptop"></a> for <strong>hands-on tutorial</strong> (on the <a href="training.galaxyproject.org/" target="_blank">Galaxy Training Material</a>)</li>  <li><a href="" class="fa fa-video-camera"></a> for <strong>video</strong> (accompanying a lecture, a tutorial or a demo)</li>  <li><a href="" class="fa fa-list-ul"></a> for <strong>shared Galaxy history</strong> (on the <a href="https://usegalaxy.eu" target="_blank">European Galaxy server</a>)</li></ul><h2 id="day-1-982021---introduction-to-galaxy">Day 1 (9.8.2021) - Introduction to Galaxy</h2><table class="table table-striped">  <thead>    <tr>      <th>Topic</th>      <th>Speaker</th>      <th>Material</th>      <th>Description</th>      <th>Duration</th>    </tr>  </thead>  <tbody>    <tr>      <td>A very short introduction to Galaxy</td>      <td>Anton Nekrutenko</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/slides.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-slideshare" target="_blank"></a> / <a href="https://youtu.be/VZoz3k5EehI" class="fa fa-video-camera" target="_blank"></a> Lecture</td>      <td><em class="small">This video will introduce the Galaxy data analysis platform, and give a short demo on how to use it.</em></td>      <td>10m</td>    </tr>    <tr>      <td>Galaxy 101</td>      <td>Anton Nekrutenko</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101/tutorial.html" class="fa fa-laptop" target="_blank"></a> / <a href="https://youtu.be/D5HgJWdfOWw" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial <br /> <em class="fa fa-list-ul">WIP</em> History</td>      <td><em class="small">This tutorial will introduce you to Galaxy. You will familiarize yourself with tools, workflows and histories. Those skills will be needed the next days.</em></td>      <td>1h</td>    </tr>    <tr>      <td>NGS data logistics</td>      <td>Anton Nekrutenko</td>      <td><a href="https://training.galaxyproject.org/training-material//topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html" class="fa fa-laptop" target="_blank"></a> / <a href="https://youtu.be/9mIL0tIfZ_o" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial <br /> <em class="fa fa-list-ul" target="_blank">WIP</em> History</td>      <td><em class="small">Learn how to manipulate and process NGS data data derived from patients infected with SARS-CoV-2. Get familiar with quality control, mapping and NGS filetypes.</em></td>      <td>1h 30m</td>    </tr>  </tbody></table><h2 id="day-2-1082021---data-upload--quality-control">Day 2 (10.8.2021) - Data Upload &amp; Quality Control</h2><table class="table table-striped">  <thead>    <tr>      <th>Topic</th>      <th>Speaker</th>      <th>Material</th>      <th>Description</th>      <th>Duration</th>    </tr>  </thead>  <tbody>    <tr>      <td> </td>      <td>All experts</td>      <td><a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing" class="fa fa-info-circle" target="_blank"></a> Info &amp; Zoom link</td>      <td><strong>Real-Time Q&amp;A session (9 am CEST)</strong></td>      <td>1h</td>    </tr>    <tr>      <td>Quality control of reads</td>      <td>Florian Heyl</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/slides.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-slideshare" target="_blank"></a> / <a href="https://youtu.be/BWonTPS4zB8" class="fa fa-video-camera" target="_blank"></a> Lecture</td>      <td><em class="small">This lecture goes over the concepts involved in assessing the quality of your sequencing data.</em></td>      <td>40m</td>    </tr>    <tr>      <td>Quality control of reads</td>      <td>Florian Heyl</td>      <td><a href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/quality-control/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://youtu.be/QJRlX2hWDKM" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial <br /> <em class="fa fa-list-ul" target="_blank">WIP</em> History</td>      <td><em class="small">In this tutorial you will get some hand-on experience performing a quality assessment on sequencing data.</em></td>      <td>1h 10m</td>    </tr>    <tr>      <td>Mapping of reads</td>      <td>Peter van Heusden</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/mapping/slides.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-slideshare" target="_blank"></a> / <a href="https://youtu.be/7FhHb8EV3EU" class="fa fa-video-camera" target="_blank"></a> Lecture</td>      <td><em class="small">This lecture covers the basic concepts involved in mapping sequencing reads to a reference genome.</em></td>      <td>10m</td>    </tr>    <tr>      <td>Mapping of reads</td>      <td>Peter van Heusden</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/mapping/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://youtu.be/1wm-62E2NkY" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial <br /> <em class="fa fa-list-ul" target="_blank">WIP</em> History</td>      <td><em class="small">In this tutorial you will map sequencing data to a reference genome, and explore the mapped reads in a genome browser.</em></td>      <td>20m</td>    </tr>    <tr>      <td>Using dataset collections</td>      <td>Anton Nekrutenko</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/collections/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://youtu.be/uZUt9XIHUQo" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial <br /> <a href="https://usegalaxy.eu/u/bgruening/h/using-dataset-collections---tutorial" class="fa fa-list-ul" target="_blank"></a> History</td>      <td><em class="small">How to manipulate large numbers of datasets at once? This will be needed to process 100 of SARS-CoV-2 samples in one go.</em></td>      <td>30m</td>    </tr>    <tr>      <td>Data cleaning workflow</td>      <td>TBC (BE)</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/human-reads-removal/tutorial.html" class="fa fa-laptop" target="_blank"></a> / <em class="fa fa-video-camera" target="_blank">WIP</em> Hands-on tutorial <br /> <em class="fa fa-list-ul" target="_blank">WIP</em> History</td>      <td><em class="small">As a first exercise in actual SARS-CoV-2 data analysis with Galaxy, this tutorial will let you perform the steps necessary to remove contaminating human reads from sequencing data of SARS-CoV-2 isolates.</em></td>      <td>1h</td>    </tr>    <tr>      <td> </td>      <td>All experts</td>      <td><a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing" class="fa fa-info-circle" target="_blank"></a> Info &amp; Zoom link</td>      <td><strong>Real-Time Q&amp;A session (4 pm CEST)</strong></td>      <td>1h</td>    </tr>  </tbody></table><h2 id="day-3-1182021---sars-cov-2-data-analysis-on-public-datasets">Day 3 (11.8.2021) - SARS-CoV-2 Data Analysis on Public Datasets</h2><table class="table table-striped">  <thead>    <tr>      <th>Topic</th>      <th>Speaker</th>      <th>Material</th>      <th>Description</th>      <th>Duration</th>    </tr>  </thead>  <tbody>    <tr>      <td> </td>      <td>All experts</td>      <td><a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing" class="fa fa-info-circle" target="_blank"></a> Info &amp; Zoom link</td>      <td><strong>Real-Time Q&amp;A session (9 am CEST)</strong></td>      <td>1h</td>    </tr>    <tr>      <td>Galaxy for SARS-CoV-2 genome surveillance projects</td>      <td>Wolfgang Maier</td>      <td><em class="fa fa-slideshare" target="_blank">WIP</em> / <em class="fa fa-video-camera" target="_blank">WIP</em> Lecture</td>      <td><em class="small">Get an overview of what day 3 has to offer: production-ready Galaxy workflows for SARS-CoV-2 sequencing data, tools you should know to automate workflow execution, and how you combine all of it to turn Galaxy into a platform for genome-surveillance.</em></td>      <td> </td>    </tr>    <tr>      <td>Variant calling, reporting, consensus building (with Galaxy GUI)</td>      <td>Wolfgang Maier</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=vnFQ2fR_fzw" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial <br /> <em class="fa fa-list-ul" target="_blank">WIP</em> History</td>      <td><em class="small">Illumina or ONT, ampliconic or WGS data? Learn how to combine the right set of Galaxy workflows to analyze the type of SARS-CoV-2 sequencing data of your choice.</em></td>      <td> </td>    </tr>    <tr>      <td>Variant calling, reporting, consensus building (with Galaxy CLI)</td>      <td>Simon Bray</td>      <td><a href="https://training.galaxyproject.org/training-material//topics/galaxy-interface/tutorials/workflow-automation/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=o39QjVnLG68" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial</td>      <td><em class="small">Learn how to use the command line to upload your SARS-CoV-2 data to a Galaxy-server and launch workflows for its analysis. Note: This first step towards automation requires the command line tool Planemo for interacting with Galaxy if you want to follow along.</em></td>      <td> </td>    </tr>    <tr>      <td>The usegalaxy.eu SARS-CoV-2 bot in action</td>      <td>Wolfgang Maier</td>      <td><em class="fa fa-video-camera" target="_blank">WIP</em> Demo</td>      <td><em class="small">See in this demo how, on usegalaxy.eu, we’ve used Planemo and Bioblend to build and operate an automated SARS-CoV-2 genome surveillance system based on the Galaxy workflows for variant calling, consensus building and reporting.</em></td>      <td> </td>    </tr>    <tr>      <td> </td>      <td>All experts</td>      <td><a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing" class="fa fa-info-circle" target="_blank"></a> Info &amp; Zoom link</td>      <td><strong>Real-Time Q&amp;A session (4 pm CEST)</strong></td>      <td>1h</td>    </tr>  </tbody></table><h2 id="day-4-1282021---visualisation--data-export">Day 4 (12.8.2021) - Visualisation &amp; Data Export</h2><table class="table table-striped">  <thead>    <tr>      <th>Topic</th>      <th>Speaker</th>      <th>Material</th>      <th>Description</th>      <th>Duration</th>    </tr>  </thead>  <tbody>    <tr>      <td> </td>      <td>All experts</td>      <td><a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing" class="fa fa-info-circle" target="_blank"></a> Info &amp; Zoom link</td>      <td><strong>Real-Time Q&amp;A session (9 am CEST)</strong></td>      <td>1h</td>    </tr>    <tr>      <td>Upload data to ENA</td>      <td>Miguel Roncoroni</td>      <td><a href="https://www.youtube.com/watch?v=POiQG-7O7rw" class="fa fa-video-camera" target="_blank"></a> Demo</td>      <td><em class="small">Learn how to submit your sequencing data to the ENA directly from Galaxy.</em></td>      <td>14m</td>    </tr>    <tr>      <td>Upload data to a local datastore</td>      <td>Wolfgang Maier</td>      <td><em class="fa fa-video-camera" target="_blank">WIP</em> Demo</td>      <td><em class="small">So you’ve used Galaxy workflows to analyze your SARS-CoV-2 samples? Learn in this tutorial how to export results to your favorite datastore.</em></td>      <td> </td>    </tr>    <tr>      <td>Introduction to viral Beacon</td>      <td>Babita Singh</td>      <td><a href="https://youtu.be/R_4yUMPk7eY" class="fa fa-video-camera" target="_blank"></a> Demo</td>      <td><em class="small">How to visualize tens of thousands of SARS-CoV-2 analysis results? Learn about the Viral Beacon project’s solution!</em></td>      <td>24m</td>    </tr>    <tr>      <td>Using and Customising ObservableHQ</td>      <td>TBC</td>      <td><em class="fa fa-video-camera" target="_blank">WIP</em> Demo</td>      <td><em class="small">In this demo you will get to know the ObservableHQ platform for interactive data visualization. You will see how covid19.galaxyproject.org uses it to build a dashboard for their SARS-CoV-2 analysis efforts and will learn how to customize this solution to fit your own purposes.</em></td>      <td> </td>    </tr>    <tr>      <td> </td>      <td>All experts</td>      <td><a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing" class="fa fa-info-circle" target="_blank"></a> Info &amp; Zoom link</td>      <td><strong>Real-Time Q&amp;A session (4 pm CEST)</strong></td>      <td>1h</td>    </tr>  </tbody></table><h2 id="optional-extra-training">Optional extra training</h2><table class="table table-striped">  <thead>    <tr>      <th>Topic</th>      <th>Speaker</th>      <th>Material</th>      <th>Description</th>      <th>Duration</th>    </tr>  </thead>  <tbody>    <tr>      <td>SRA Aligned Read Formats to Speed Up SARS-CoV-2 data Analysis</td>      <td>Jonathan Trow</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/ncbi-sarf/slides.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-slideshare" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=siLP71B9gm4" class="fa fa-video-camera" target="_blank"></a> Lecture</td>      <td><em class="small">This lecture will introduce the SRA Aligned Read format available in the cloud from SRA, as well as some accompanying metadata that can help you search and filter the data. This sessions is aimed specifically at SARS-CoV-2 runs in SRA.</em></td>      <td>15m</td>    </tr>    <tr>      <td>SRA Aligned Read Formats to Speed Up SARS-CoV-2 data Analysis</td>      <td>Jonathan Trow</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=ogu-NBTP-DM" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial</td>      <td><em class="small">This tutorials will walk you through accessing and using SRA Aligned read format in Galaxy.</em></td>      <td>40m</td>    </tr>    <tr>      <td>Assembly: Unicycler assembly of SARS-CoV-2 genome</td>      <td>Cristóbal Gallardo</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/assembly/tutorials/assembly-with-preprocessing/slides.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> Lecture</td>      <td> </td>      <td> </td>    </tr>    <tr>      <td>Assembly: Unicycler assembly of SARS-CoV-2 genome</td>      <td>Cristóbal Gallardo</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=jNFLYhjgJPs" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial</td>      <td> </td>      <td>25m</td>    </tr>    <tr>      <td>Pandemics Research using Mass Spectrometry</td>      <td>Timothy J. Griffin, Subina Mehta, Andrew Rajczewski, Pratik Jagtap</td>      <td><a href="https://drive.google.com/file/d/1anBPmGRWEVp9pBLZ_JInWV7iWrBpyCDs/view?usp=sharing?utm_source=smorgasbord&amp;utm_medium=website&amp;utm_campaign=gcc2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=CI35gTmZoqM" class="fa fa-video-camera" target="_blank"></a> Demo</td>      <td> </td>      <td>35m</td>    </tr>    <tr>      <td>Running Workflows from the Command Line using Planemo</td>      <td>Simon Bray</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-automation/tutorial.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-laptop" target="_blank"></a> / <a href="https://www.youtube.com/watch?v=o39QjVnLG68" class="fa fa-video-camera" target="_blank"></a> Hands-on tutorial</td>      <td><em class="small">This is a more technical tutorial which will teach you how to run workflows from the command line and scale your analyses. If you are on Windows, you will need WSL/WSL2 setup before this session.</em></td>      <td>30m</td>    </tr>    <tr>      <td>Scripting Galaxy using the API and BioBlend</td>      <td>TBC</td>      <td><a href="https://training.galaxyproject.org/training-material/topics/dev/tutorials/bioblend-api/slides.html?utm_source=usegalaxyeu&amp;utm_medium=website&amp;utm_campaign=covid2021" class="fa fa-slideshare" target="_blank"></a></td>      <td> </td>      <td> </td>    </tr>    <tr>      <td>What you can do with SARS-COV-2 data: Case studies</td>      <td>Andrew Page</td>      <td><a href="https://www.youtube.com/watch?v=R4EoTEiAQNE" class="fa fa-video-camera" target="_blank"></a></td>      <td> </td>      <td>37m</td>    </tr>  </tbody></table><h1 id="logistics">Logistics</h1><h2 id="content-delivery">Content delivery</h2>This is a global workshop delivered asynchronously. In practice, this means that you will have training materials available to explore them at <strong>your own pace</strong>, without any time constraints:\n\n<ul>  <li><strong>Lectures</strong>: pre-recorded videos (<a href="" class="fa fa-video-camera"></a>) with the theoretical explanation of the lesson, supported by slide decks (<a href="" class="fa fa-slideshare"></a>).</li>  <li>    <strong>Hands-on tutorials</strong> (<a href="" class="fa fa-laptop"></a>): a step-by-step explanation, including all the required information, to perform a data analysis, often available also as pre-recorded video (<a href="" class="fa fa-video-camera"></a>).\n\n    Most of the tutorials are developed by the <a href="http://training.galaxyproject.org/" target="_blank">Galaxy Training Network</a>. A feedback form is available at the bottom of each tutorial page. Please fill it out, it helps us to value and improve the tutorials.\n\n  </li>  <li><strong>Histories</strong>: shared Galaxy history (<a href="" class="fa fa-list-ul"></a>), on the <a href="https://usegalaxy.eu" target="_blank">European Galaxy server</a>, with all that you need to reproduce what is shown in the hands-on part.</li>  <li><strong>Demo</strong>: pre-recorded videos (<a href="" class="fa fa-video-camera"></a>) demonstrating a technical point or a nice feature.</li></ul>Most of the material is available already, and they will all stay <strong>available after the workshop</strong>. Most of the material have been developed by a community of people via the <a href="training.galaxyproject.org/" target="_blank">Galaxy Training Network</a>. Some videos were recorded for different previous events, e.g. <a href="https://shiltemann.github.io/global-galaxy-course/" target="_blank">GTN Smörgåsbord</a> or <a href="https://galaxyproject.org/events/gcc2021/training/" target="_blank">GCC2021 Training Week</a>, and the captions were manually-curated by several community members.\n\nWhenever you’re ready to get started, you can access the material by clicking on the different icons in the program!\n\n<h2 id="doing-the-tutorials---technical-requirements">Doing the tutorials - Technical requirements</h2>Some of you have asked about the technical requirements. You don’t need a specific operating system or software installed, all you need is a browser and internet connection.\n\nTo run the tutorials, you will need a <strong>Galaxy account</strong>. We recommend you to:\n\n<ul>  <li>    Register and use <a href="https://usegalaxy.eu" target="_blank">European Galaxy server</a>\n\n    You will receive an email to confirm your account before you can start using Galaxy.\n\n  </li>  <li>    Join the <a href="https://usegalaxy.eu/join-training/sars_cov2_2021/">training event group</a>, it will make your jobs go faster!\n\n  </li></ul><h2 id="support--communication-channels">Support &amp; Communication channels</h2>Should you have any questions, the instructors will be available in chat. We will use the Slack space of the Galaxy Training Network. Depending on your location you might need to use a VPN, so <strong>please make sure that you can join Slack before the workshop</strong>.\n\nOnce you are in, you will see different channels (#general, #covid2021-day- for the different days) but also #random, #social. Pass by and say hi to your colleagues! Every day we will have an icebreaker question for the #social channel.\n\nWhen asking a question:\n\n<ul>  <li>Ask in the appropriate place:    <ul>      <li>#general for general issues</li>      <li>#event-covid2021-day-1 for day 1 (Introduction to Galaxy)</li>      <li>#event-covid2021-day-2 for day 2 (Data Upload &amp; Quality Control)</li>      <li>#event-covid2021-day-3 for day 3 (SARS-CoV-2 Data Analysis on Public Datasets)</li>      <li>#event-covid2021-day-4 for day 4 (Visualisation &amp; Data Export)</li>    </ul>  </li>  <li>Use threads.</li>  <li>Say which server you’re using.</li>  <li>Share all of the details (What did the tool say? What was the error? Did you see more information in the bug-report icon?)</li></ul>During the week of the workshop, the instructors will be there to reply to your questions. Please be aware of the <strong>time zones</strong>, the instructors are scattered all over the world and sometimes you may have to be patient to get a reply.\n\nReal-time Q&amp;A sessions will be run on days 2, 3 and 4 (9 am and 4 pm CEST). Find the details to join these sessions and register by adding your name to the attendees list in the <a href="https://docs.google.com/document/d/1SIcl7BPDGCV0D-AA_S008qVYaH8id4_ibD6iTVx6gYY/edit?usp=sharing">dedicated document</a>.\n\n<h2 id="certificates">Certificates</h2>If you need a certificate, you can request it at the end of the workshop. Please make sure to keep all the work over the 4 days, stay active in the discussions and fill out the <a href="https://forms.gle/m8NRg7WrgGwnCEGQA">final survey</a>.\n\n<h1 id="code-of-conduct">Code of Conduct</h1>Everyone is expected to abide by the <a href="https://galaxyproject.org/community/coc/">Code of Conduct (CoC)</a> to make this environment welcoming and friendly for everyone.\n\n<h1 id="instructors--helpers">Instructors &amp; helpers</h1><table class="table table-striped">  <thead>    <tr>      <th>Name</th>      <th>Location</th>    </tr>  </thead>  <tbody>    <tr>      <td>Björn Grüning</td>      <td>Germany</td>    </tr>    <tr>      <td>Wolfgang Maier</td>      <td>Germany</td>    </tr>    <tr>      <td>Anton Nekrutenko</td>      <td>USA</td>    </tr>    <tr>      <td>Andrew Page</td>      <td>UK</td>    </tr>    <tr>      <td>Peter van Heusden</td>      <td>South Africa</td>    </tr>    <tr>      <td>Simon Bray</td>      <td>Germany</td>    </tr>    <tr>      <td>Erik Hjerde</td>      <td>Norway</td>    </tr>    <tr>      <td>Annbjørg Barbakken</td>      <td>Norway</td>    </tr>    <tr>      <td>Kjell Petersen</td>      <td>Norway</td>    </tr>    <tr>      <td>Steven Morgan</td>      <td>Australia</td>    </tr>    <tr>      <td>Gareth Price</td>      <td>Australia</td>    </tr>    <tr>      <td>Anna Syme</td>      <td>Australia</td>    </tr>    <tr>      <td>Igor Makunin</td>      <td>Australia</td>    </tr>    <tr>      <td>Valentine Murigneux</td>      <td>Australia</td>    </tr>    <tr>      <td>Michael Thang</td>      <td>Australia</td>    </tr>  </tbody></table>
LOCATION:online
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:SARS-CoV-2 Data Analysis and Monitoring with Galaxy
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
