<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe | New and updated CNV and Variant Calling tools</title>
  <meta property="og:title" content="New and updated CNV and Variant Calling tools" />
  <meta property="og:description" content="
Copy number variation (CNV) is a type of structural vari..." />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/posts/2021/01/30/cnv-and-vc-updates/">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>


    
    
    

  <body class="location-pasteurfreiburgerasmusmcelixir-itbelgiumgenouest">
    <div id="wrap">
      <div id="main">
        <div class="navbar">

  <div class="container">
    
    

    
    
    

    

    <div class="navbar-header">
      <div class="navbar-brand">
        
        <img alt="galaxy eu logo" src="/assets/media/galaxy-eu-logo.inv.svg" style="height: 20px"/>
        
      </div>

      <a class="navbar-brand" href="/" style="padding-left: 6px">
        Galaxy Europe
      </a>
      
        <span class="navbar-brand" style="color: white">
        /
        </span>
        <a class="navbar-brand" href="https://galaxyproject.eu/%5B%22pasteur%22,%20%22freiburg%22,%20%22erasmusmc%22,%20%22elixir-it%22,%20%22belgium%22,%20%22genouest%22%5D">
          
        </a>
      
    </div>

      <div class="navbar-collapse collapse">
        <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="true">Member Sites <span class="caret"></span></a>
             <ul class="dropdown-menu">
              
                <li>
                  <a class="page-link" href="/freiburg/">Freiburg</a>
                </li>
              
                <li>
                  <a class="page-link" href="/erasmusmc/">Erasmus MC</a>
                </li>
              
                <li>
                  <a class="page-link" href="/belgium/">VIB</a>
                </li>
              
                <li>
                  <a class="page-link" href="/pasteur/">Pasteur</a>
                </li>
              
                <li>
                  <a class="page-link" href="/genouest/">GenOuest</a>
                </li>
              
                <li>
                  <a class="page-link" href="/elixir-it/">ELIXIR-IT/Laniakea</a>
                </li>
              
                <li>
                  <a class="page-link" href="/ifb/">ELIXIR-FR/IFB</a>
                </li>
              
             </ul>
           </li>

          
          
            <li>
              <a class="page-link" href="/news">News</a>
            </li>
          
            <li>
              <a class="page-link" href="/events">Events</a>
            </li>
          
            <li>
              <a class="page-link" href="/people">People</a>
            </li>
          
            <li>
              <a class="page-link" href="/citations">Citations</a>
            </li>
          
            <li>
              <a class="page-link" href="/tools">Tools</a>
            </li>
          
            <li>
              <a class="page-link" href="/about">About</a>
            </li>
          
          
        </ul>
      </div>
  </div>
</div>

        <div class="container" id="maincontainer">
          




<section id="news-page">
  <div class="row news-header-row">
    
    <a class="col-xs-1 text-center" href="/posts/2021/01/28/tiaas_Lachlan/">
      «
    </a>
    
    <div class="col-xs-10 ">
      <div class="text-center">
        <h2 class="title">New and updated CNV and Variant Calling tools</h2>
      </div>
      <hr />
    </div>
    
    <a class="col-xs-1 text-center" href="/posts/2021/01/30/tool-update/">
      »
    </a>
    
  </div>
  <div class="row post-metadata">
    <div class="col-md-8 col-md-offset-2">
      <div class="row">
        <div class="col-md-6">
            









By: <a href="/people/#wm75">
    <img src="https://avatars.githubusercontent.com/wm75" style="height: 2em">
    Wolfgang Maier
</a>





















































































        </div>
        <div class="col-md-6 text-right">
          <div>
            <span class="post-date">
              Jan 30, 2021
            </span>

            

            <br />
            <span class="post-tags">
                <span class="label label-success"></span>
              
                <span class="label label-info">tools</span>
              
              
            </span>

            
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="row">
    <div class="col-md-8 col-md-offset-2">
      <div class="news-content">
        <table><tr><td>
Copy number variation (CNV) is a type of structural variation, specifically, it is a type of duplication or deletion events.
During the last month the Galaxy community has cooperated with the CNV community to bring a variety of the best CNV tools to the 
Galaxy ecosystem. In addition we updated the variant calling tools, specifically to also cope with long read sequencing technologies.
</td><td>
<img src="/assets/media/humman_variations_community_icon.png" alt="ELIXIR CNV community" />
</td></tr></table>

<p>In the following you will find a list of a few highlights.
We could like to thank all contributors, especially Nathan Roach, Niko Pinter, Stephan Flemming and the <a href="https://elixir-europe.org/communities/hcnv">ELIXIR CNV commuity</a>.</p>

<hr />

<hr />
<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/strelka_somatic/strelka_somatic/">Strelka2</a></strong></p>

<p>is a fast and accurate small variant caller optimized for analysis of germline variation in small cohorts
and somatic variation in tumor/normal sample pairs.
<a href="(https://github.com/Illumina/strelka)">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/strelka_somatic/strelka_somatic/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/irissv/irissv/">Iris</a></strong></p>

<p>is a module which corrects the sequences of structural variant calls (currently only insertions).
<a href="https://github.com/mkirsche/Iris">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/irissv/irissv/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/jasminesv/jasminesv/">JASMINE</a></strong></p>

<p>is tool is used to merge structural variants (SVs) across samples.
<a href="https://github.com/mkirsche/Jasmine">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/jasminesv/jasminesv/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/lumpy_sv/lumpy_sv/">Lumpy</a></strong></p>

<p>is probabilistic framework for structural variant discovery.
<a href="https://github.com/arq5x/lumpy-sv">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/lumpy_sv/lumpy_sv/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/delly_call/delly_call/">Delly</a></strong></p>

<p>is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications,
inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data.
<a href="https://github.com/dellytools/delly">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/delly_call/delly_call/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/sansa_annotate/sansa_annotate/">Sansa</a></strong></p>

<p>stands for Structural variant (SV) annotation and this is what is does.
<a href="https://github.com/dellytools/sansa">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/sansa_annotate/sansa_annotate/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/sniffles/sniffles/">Sniffles</a></strong></p>

<p>is a structural variation caller using third generation sequencing (PacBio or Oxford Nanopore). It detects all types of SVs (10bp+)
using evidence from split-read alignments, high-mismatch regions, and coverage analysis.
<a href="https://github.com/fritzsedlazeck/Sniffles">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/sniffles/sniffles/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/cutesv/cutesv/">cuteSV</a></strong></p>

<p>is a sensitive, fast and scalable long-read-based SV detection approach. cuteSV uses tailored methods to
collect the signatures of various types of SVs and employs a clustering-and-refinement method
to analyze the signatures to implement sensitive SV detection.
<a href="https://github.com/tjiangHIT/cuteSV">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/cutesv/cutesv/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/control_freec/control_freec">Control-FREEC</a></strong></p>

<p>is a tool for detection of copy-number changes and allelic imbalances (including LOH) using deep-sequencing data.
<a href="http://boevalab.inf.ethz.ch/FREEC/">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/control_freec/control_freec">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant_pipeline/medaka_variant_pipeline/">Medaka</a></strong></p>

<p>is a tool to create consensus sequences and variant calls from nanopore sequencing data
using neural networks applied a pileup of individual sequencing reads against a draft assembly.
<a href="https://github.com/nanoporetech/medaka">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant_pipeline/medaka_variant_pipeline/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/gatk4_mutect2/gatk4_mutect2/">GATK4 Mutect2</a></strong></p>

<p>calls somatic short mutations via local assembly of haplotypes. Short mutations include single nucleotide (SNA) and insertion and deletion (indel)
alterations. The caller uses a Bayesian somatic genotyping model.
<a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/gatk4_mutect2/gatk4_mutect2/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/basil/basil/">BASIL</a></strong></p>

<p>is a method to detect breakpoints for structural variants (including insertion breakpoints) from aligned paired HTS reads in BAM format.
<a href="https://github.com/seqan/anise_basil">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/basil/basil/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/vcfanno/vcfanno/">vcfanno</a></strong></p>

<p>allows you to quickly annotate your VCF with any number of INFO fields from any number of VCFs or BED files.
<a href="https://github.com/brentp/vcfanno">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/vcfanno/vcfanno/">[Galaxy tool]</a></p>

<hr />

<p><strong><a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/artic_minion/artic_minion/">ARTIC minion</a></strong></p>

<p>builds consensus sequence and call variants from amplicon-based nanopore sequence data.
<a href="https://github.com/artic-network/fieldbioinformatics">[Project link]</a>
<a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/artic_minion/artic_minion/">[Galaxy tool]</a></p>

<hr />


      </div>
    </div>
  </div>


	<div class="col-md-2">
	</div>
<div class="row">
	<div class="col-md-8 col-sm-12">

<div id="supported">
    
  <h3>Supported by</h3>
    
  <div class="row eu-image-box">
      
      <div>
      
          
            <a href="https://usegalaxy-eu.github.io" target="_blank">
              <img alt="European Galaxy Team Logo" src="/assets/media/galaxy-eu.png" class="img-responsive eu-image" />
            </a>
          
      </div>
      
      <div>
      
          
            <a href="https://www.miracum.org/" target="_blank">
              <img alt="Miracum" src="/assets/media/miracum.png" class="img-responsive eu-image"/>
            </a>
          
      </div>
      
  </div>
</div>


	</div>
	<div class="col-md-2">
	</div>
</div>

</section>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the member sites in this repository contribute to the European Galaxy Project.
For <strong>acknowledgement</strong>, please refer to the <a href="/about">About</a> section.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a> unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/_posts/2021-01-30-cnv-and-vc-updates.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:galaxy@informatik.uni-freiburg.de">galaxy@informatik.uni-freiburg.de</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
