<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe | UseGalaxy.eu Tool Updates for 2021-04-14</title>
  <meta property="og:title" content="UseGalaxy.eu Tool Updates for 2021-04-14" />
  <meta property="og:description" content="On 2021-04-14, the tools on UseGalaxy.eu were updated by ..." />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/posts/2021/04/14/tool-update/plain.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          




<section id="news-page">
  <div class="row news-header-row">
    
    <a class="col-xs-1 text-center" href="/posts/2021/04/12/galactic-news/plain.html">
      «
    </a>
    
    <div class="col-xs-10 ">
      <div class="text-center">
        <h2 class="title">UseGalaxy.eu Tool Updates for 2021-04-14</h2>
      </div>
      <hr />
    </div>
    
    <a class="col-xs-1 text-center" href="/posts/2021/04/15/gcc-abstracts/plain.html">
      »
    </a>
    
  </div>
  <div class="row post-metadata">
    <div class="col-md-8 col-md-offset-2">
      <div class="row">
        <div class="col-md-6">
            

        </div>
        <div class="col-md-6 text-right">
          <div>
            <span class="post-date">
              Apr 14, 2021
            </span>

            

            <br />
            <span class="post-tags">
                <span class="label label-success">Freiburg</span>
              
                <span class="label label-info">tools</span>
              
              
            </span>

            
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="row">
    <div class="col-md-8 col-md-offset-2">
      <div class="news-content">
        <p>On 2021-04-14, the tools on UseGalaxy.eu were updated by our automated tool update and installation process in <a href="https://build.galaxyproject.eu/job/usegalaxy-eu/job/install-tools/#270/">Jenkins Build #270</a></p>

<h2 id="machine-learning">Machine Learning</h2>

<ul>
  <li>keras_batch_models was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_batch_models/0dc3027df8f4">0dc3027df8f4</a></li>
  <li>keras_batch_models was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_batch_models/f5e7df4f7975">f5e7df4f7975</a></li>
  <li>keras_model_builder was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_model_builder/c51571d0a0b1">c51571d0a0b1</a></li>
  <li>keras_model_builder was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_model_builder/e3b420d0b71a">e3b420d0b71a</a></li>
  <li>keras_model_config was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_model_config/2d8478144bb2">2d8478144bb2</a></li>
  <li>keras_model_config was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_model_config/64bbfa592868">64bbfa592868</a></li>
  <li>keras_train_and_eval was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/keras_train_and_eval/785dd890e27d">785dd890e27d</a></li>
  <li>ml_visualization_ex was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/ml_visualization_ex/14bd6d59650d">14bd6d59650d</a></li>
  <li>model_prediction was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/model_prediction/22f9cbcf1582">22f9cbcf1582</a></li>
  <li>sklearn_build_pipeline was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_build_pipeline/664666bd14ec">664666bd14ec</a></li>
  <li>sklearn_clf_metrics was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_clf_metrics/40e31fb88836">40e31fb88836</a></li>
  <li>sklearn_data_preprocess was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_data_preprocess/b75cae00f980">b75cae00f980</a></li>
  <li>sklearn_discriminant_classifier was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_discriminant_classifier/420a4bf99244">420a4bf99244</a></li>
  <li>sklearn_ensemble was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_ensemble/a98fdfcd9f5f">a98fdfcd9f5f</a></li>
  <li>sklearn_estimator_attributes was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_estimator_attributes/74de84506e74">74de84506e74</a></li>
  <li>sklearn_feature_selection was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_feature_selection/1d20e0dce176">1d20e0dce176</a></li>
  <li>sklearn_fitted_model_eval was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_fitted_model_eval/e00c1c9d206b">e00c1c9d206b</a></li>
  <li>sklearn_generalized_linear was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_generalized_linear/616a241c5b37">616a241c5b37</a></li>
  <li>sklearn_lightgbm was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_lightgbm/aff86cee673c">aff86cee673c</a></li>
  <li>sklearn_model_fit was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_model_fit/5dc03d8e137e">5dc03d8e137e</a></li>
  <li>sklearn_model_validation was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_model_validation/de360b57a5ab">de360b57a5ab</a></li>
  <li>sklearn_nn_classifier was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_nn_classifier/34d31bd995e9">34d31bd995e9</a></li>
  <li>sklearn_numeric_clustering was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_numeric_clustering/e7f047a9dca9">e7f047a9dca9</a></li>
  <li>sklearn_pairwise_metrics was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_pairwise_metrics/cd4328a02798">cd4328a02798</a></li>
  <li>sklearn_pca was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_pca/e36ab18cbaca">e36ab18cbaca</a></li>
  <li>sklearn_regression_metrics was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_regression_metrics/9c8ede554b7c">9c8ede554b7c</a></li>
  <li>sklearn_sample_generator was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_sample_generator/999e07f0a9fa">999e07f0a9fa</a></li>
  <li>sklearn_searchcv was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_searchcv/586e68c83df2">586e68c83df2</a></li>
  <li>sklearn_stacking_ensemble_models was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_stacking_ensemble_models/2b679bb676be">2b679bb676be</a></li>
  <li>sklearn_svm_classifier was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_svm_classifier/a5665e1b06b0">a5665e1b06b0</a></li>
  <li>sklearn_to_categorical was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_to_categorical/59e8b4328c82">59e8b4328c82</a></li>
  <li>sklearn_train_test_eval was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_train_test_eval/a9e0b963b7bb">a9e0b963b7bb</a></li>
  <li>sklearn_train_test_split was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_train_test_split/13b9ac5d277c">13b9ac5d277c</a></li>
</ul>

<h2 id="statistics">Statistics</h2>

<ul>
  <li>scipy_sparse was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/scipy_sparse/318484f56b6a">318484f56b6a</a></li>
</ul>

<h2 id="chemicaltoolbox">ChemicalToolBox</h2>

<ul>
  <li>sucos_clustering was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sucos_clustering/b8725fec8c7b">b8725fec8c7b</a></li>
  <li>sucos_docking_scoring was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sucos_docking_scoring/4f1896782f7c">4f1896782f7c</a></li>
  <li>sucos_max_score was updated to <a href="https://toolshed.g2.bx.psu.edu/view/bgruening/sucos_max_score/9b48456a96fe">9b48456a96fe</a></li>
</ul>

<h2 id="convert-formats">Convert Formats</h2>

<ul>
  <li>thermo_raw_file_converter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/thermo_raw_file_converter/b9b385097f13">b9b385097f13</a></li>
</ul>

<h2 id="proteomics">Proteomics</h2>

<ul>
  <li>openms_openpepxl was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_openpepxl/40cc19c0dbd6">40cc19c0dbd6</a></li>
  <li>openms_openpepxl was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_openpepxl/a2a842b00f9c">a2a842b00f9c</a></li>
  <li>openms_openpepxl was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_openpepxl/fc36cfd19834">fc36cfd19834</a></li>
  <li>openms_percolatoradapter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_percolatoradapter/0fc9ae55bcfc">0fc9ae55bcfc</a></li>
  <li>openms_percolatoradapter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_percolatoradapter/e7881a82b56d">e7881a82b56d</a></li>
  <li>openms_xfdr was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_xfdr/81402a5e4173">81402a5e4173</a></li>
  <li>openms_xfdr was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/openms_xfdr/9a6e3bb0f358">9a6e3bb0f358</a></li>
  <li>peptideshaker was updated to <a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/peptideshaker/07ff622ec007">07ff622ec007</a></li>
</ul>

<h2 id="hca_sc_scanpy_tools">hca_sc_scanpy_tools</h2>

<ul>
  <li>anndata_ops was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/anndata_ops/e4bb4666449e">e4bb4666449e</a></li>
  <li>scanpy_compute_graph was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_compute_graph/1dc0ea7a7e62">1dc0ea7a7e62</a></li>
  <li>scanpy_filter_cells was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_filter_cells/df56c78c5a8e">df56c78c5a8e</a></li>
  <li>scanpy_filter_genes was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_filter_genes/da2808112bb4">da2808112bb4</a></li>
  <li>scanpy_find_cluster was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_find_cluster/1e820072e8f1">1e820072e8f1</a></li>
  <li>scanpy_find_markers was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_find_markers/a6a896082cd7">a6a896082cd7</a></li>
  <li>scanpy_find_variable_genes was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_find_variable_genes/144c405d3ca4">144c405d3ca4</a></li>
  <li>scanpy_integrate_bbknn was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_integrate_bbknn/89eef46fce58">89eef46fce58</a></li>
  <li>scanpy_integrate_combat was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_integrate_combat/7322e7583847">7322e7583847</a></li>
  <li>scanpy_integrate_harmony was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_integrate_harmony/3f61bf17ca50">3f61bf17ca50</a></li>
  <li>scanpy_integrate_mnn was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_integrate_mnn/b202e078a83f">b202e078a83f</a></li>
  <li>scanpy_normalise_data was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_normalise_data/28ab1cdd0cf5">28ab1cdd0cf5</a></li>
  <li>scanpy_parameter_iterator was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_parameter_iterator/328e7846309d">328e7846309d</a></li>
  <li>scanpy_plot_embed was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_plot_embed/7c601934578b">7c601934578b</a></li>
  <li>scanpy_plot_trajectory was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_plot_trajectory/2fe4b4cba312">2fe4b4cba312</a></li>
  <li>scanpy_read_10x was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_read_10x/1b84ae7595ed">1b84ae7595ed</a></li>
  <li>scanpy_regress_variable was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_regress_variable/1ad1d9713563">1ad1d9713563</a></li>
  <li>scanpy_run_diffmap was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_diffmap/6586bfb21f41">6586bfb21f41</a></li>
  <li>scanpy_run_dpt was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_dpt/c5a313b4337c">c5a313b4337c</a></li>
  <li>scanpy_run_fdg was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_fdg/14866357fbec">14866357fbec</a></li>
  <li>scanpy_run_paga was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_paga/0488639e0e8a">0488639e0e8a</a></li>
  <li>scanpy_run_pca was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_pca/da13741428bf">da13741428bf</a></li>
  <li>scanpy_run_tsne was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_tsne/2c6340dad187">2c6340dad187</a></li>
  <li>scanpy_run_umap was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_run_umap/3a7b97ddf3ff">3a7b97ddf3ff</a></li>
  <li>scanpy_scale_data was updated to <a href="https://toolshed.g2.bx.psu.edu/view/ebi-gxa/scanpy_scale_data/796cd21bb304">796cd21bb304</a></li>
</ul>

<h2 id="multiple-alignments">Multiple Alignments</h2>

<ul>
  <li>chromeister was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/chromeister/e483be1014b6">e483be1014b6</a></li>
</ul>

<h2 id="metagenomic-analysis">Metagenomic Analysis</h2>

<ul>
  <li>jellyfish was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/jellyfish/b4038a4d48f5">b4038a4d48f5</a></li>
</ul>

<h2 id="single-cell">Single-cell</h2>

<ul>
  <li>scanpy_cluster_reduce_dimension was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_cluster_reduce_dimension/3e4e1cf73d98">3e4e1cf73d98</a></li>
  <li>scanpy_filter was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_filter/5511647110ff">5511647110ff</a></li>
  <li>scanpy_inspect was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_inspect/e4c99c83dfbf">e4c99c83dfbf</a></li>
  <li>scanpy_normalize was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_normalize/51d945eab1e1">51d945eab1e1</a></li>
  <li>scanpy_plot was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_plot/9a169729c9f9">9a169729c9f9</a></li>
  <li>scanpy_remove_confounders was updated to <a href="https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_remove_confounders/c2ef03559dc6">c2ef03559dc6</a></li>
</ul>


      </div>
    </div>
  </div>


<div id="supported">
    
    <h1>Supported by</h1>
    
    <div class="row eu-image-box">
        
        
        
        
        
        <div class="col-md-6">
            
            
            <a href="https://www.denbi.de" target="_blank">
                <img alt="de.NBI logo" src="/assets/media/deNBI_Logo_rgb.png" class="img-responsive eu-image" />
            </a>
            
        </div>
        
        
        
        
        <div class="col-md-6">
            
            
            <a href="https://www.elixir-europe.org" target="_blank">
                <img alt="ELIXIR logo" src="/assets/media/elixir_logo.png" class="img-responsive eu-image" />
            </a>
            
        </div>
        
    </div>
</div>


</section>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>The <a href="/freiburg/people">Freiburg Galaxy Team</a> is headed by Björn Grüning
and is part of the <a href="http://www.bioinf.uni-freiburg.de">Bioinformatics Group
Freiburg</a>, headed by Prof. Dr. Rolf
Backofen.</p>

<p>The team is part of <a href="https://www.denbi.de">de.NBI</a> and
<a href="https://www.elixir-europe.org">ELIXIR</a> and largely funded by
<a href="http://dfg.de/">DFG</a> and <a href="https://www.bmbf.de">BMBF</a>. All content on
this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>,
unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/_posts_plain/2021-04-14-tool-update.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.eu">contact@usegalaxy.eu</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
